src/robusta/core/sinks/webhook/webhook_sink.py

```diff
@@ -0,0 +1,71 @@
+import textwrap
+
+import requests
+from typing import List
+
+from .webhook_sink_params import WebhookSinkConfigWrapper
+from ..transformer import Transformer
+from ...reporting import HeaderBlock, ListBlock, JsonBlock, KubernetesDiffBlock, MarkdownBlock
+from ...reporting.base import Finding, BaseBlock
+from ..sink_base import SinkBase
+
+
+class WebhookSink(SinkBase):
+    def __init__(self, sink_config: WebhookSinkConfigWrapper, cluster_name: str):
+        super().__init__(sink_config.webhook_sink)
+        self.cluster_name = cluster_name
+        self.url = sink_config.webhook_sink.url
+        self.size_limit = sink_config.webhook_sink.size_limit
+
+    def write_finding(self, finding: Finding, platform_enabled: bool):
+        message_lines: List[str] = [finding.title]
+        if platform_enabled:
+            message_lines.append(f"Investigate: {finding.investigate_uri}")
+        message_lines.append(f"Source: {self.cluster_name}")
+        message_lines.append(finding.description)
+
+        message = ""
+
+        for enrichment in finding.enrichments:
+            for block in enrichment.blocks:
+                message_lines.extend(self.__to_unformatted_text(block))
+
+        for line in [line for line in message_lines if line]:
+            wrapped = textwrap.dedent(
+                f"""
+                {line}
+                """
+            )
+            while len(message) + len(wrapped) >= self.size_limit:
```

Comment:
I think an if will be less confusing here

Reply:
Will breaking it to 2 lines make it better?
non_empty_lines = [line for line in message_lines if line]
for line in non_empty_lines: ...

Reply:
Good catch. Fixed